static/js/components/subject_page/block.tsx

```diff
@@ -404,22 +404,21 @@ function renderTiles(
           ></GaugeTile>
         );
       case "DONUT":
+        // TODO(dwnoble): Add footnote, svg chart height, and stat var spec support to this tile
         return (
-          <DonutTile
+          <datacommons-pie
```

Comment:
To confirm, is the plan to eventually migrate everything in the website repo (minus visualization tools) to web components?

We might want to keep using `DonutTile` instead of `datacommons-pie` until all the parity/refactoring work is done. This way we don't lose current footnote or chart height functionality.

Reply:
That makes sense - reverted this to DonutTile for now